diesel/src/pg/types/money.rs

```diff
@@ -39,6 +40,40 @@ impl ToSql<types::Money, Pg> for PgMoney {
     }
 }
 
+/// # Panics
+/// will `panic!` on overflow
+impl Add for PgMoney {
+    type Output = Self;
+    fn add(self, rhs: PgMoney) -> Self::Output {
+        self.0.checked_add(rhs.0).map(PgMoney).expect("overflow adding money amounts")
+    }
+}
+
+/// # Panics
+/// will `panic!` on overflow
+impl AddAssign for PgMoney {
+    fn add_assign(&mut self, rhs: PgMoney) {
+        self.0 += rhs.0
```

Comment:
Wait, does this _actually_ panic? It calls `i64::add_assign`, right? I don't think this is a checked add in release mode!